docs/i18n.md

```diff
@@ -0,0 +1,93 @@
+# Internationalization (i18n)
+Stratos UI supports i18n via Angular's [$translate service](https://github.com/angular-translate/angular-translate). Dates and times are handled by [Moment](https://momentjs.com/)'s i18n process.
+
+The locale shown will at first be selected from the browser. Any change of locale by the user will be retained within the browser's session. 
+
+The source for all translations can be found in the relevant plugin directories i18n folder. For examples
+
+```
+|-- i18n
+|   |-- en_US
+```
+
+or
+
+```
+|-- frontend
+|   |-- i18n
+|       |-- en_US
+```
+
+Within each locale will be one or more json files. All json files within a locale will be combined and served as i18n/locale-<locale>.json.
+
+## Supported Locales
+At the moment the only supported locale is en-US. 
```

Comment:
The Stratos UI Console is currently provided with a single locale - US English (en-US). Other locales can easily be added as needed.